client/src/stylings/aboutUs.css

```diff
@@ -0,0 +1,97 @@
+.pageDiv {
+  display: flex;
+  flex-direction: column;
+  font-family: var(font-family);
```

Comment:
Great use of variable! It's a useful and powerful feature for consistency in CSS